src/transformers/pipelines/text_to_audio.py

```diff
@@ -148,10 +148,9 @@ def _forward(self, model_inputs, **kwargs):
         else:
             if len(generate_kwargs):
                 raise ValueError(
-                    f"""You're using the `TextToAudioPipeline` with a forward-only model, but `generate_kwargs` is non empty.
-                                 For forward-only TTA models, please use `forward_params` instead of of
-                                 `generate_kwargs`. For reference, here are the `generate_kwargs` used here:
-                                 {generate_kwargs.keys()}"""
+                    "You're using the `TextToAudioPipeline` with a forward-only model, but `generate_kwargs` is non "
+                    "empty. For forward-only TTA models, please use `forward_params` instead of `generate_kwargs`. "
+                    f"For reference, here are the `generate_kwargs` used here: {generate_kwargs.keys()}"
```

Comment:
```suggestion
                    f"For reference, the `generate_kwargs` used here are: {generate_kwargs.keys()}"
```
Tiny style nit to avoid repeating "here"